src/components/tl/ucp/tl_ucp_coll.h

```diff
@@ -71,8 +71,15 @@ extern const char
         }                                                                      \
     } while (0)
 
+
+enum ucc_tl_ucp_task_flags {
+    /*indicates whether subset field of tl_ucp_task is set*/
+    UCC_TL_UCP_TASK_FLAG_SUBSET = UCC_BIT(0),
```

Comment:
do we need this flag? the original idea was that subset is set to EP_MAP_FULL by default. So if collective supports it, then it always uses "subset" which evaluates to no-op for FULL. Otherwise works as expected.

Reply:
it's used to identify if subset is already set and shouldn't be changed. For example, in service collective.